src/index.ts

```diff
@@ -0,0 +1,97 @@
+import {createServer, IncomingMessage, ServerResponse} from 'http'
+import * as fs from 'fs'
+import 'body-parser'
+
+const PATH: string = './file/superFile.txt'
+const HOSTNAME: string = '127.0.0.1'
+const PORT: number = 3000
+
+const server = createServer((req: IncomingMessage, res: ServerResponse) => {
+    if (req.url === '/') {
```

Comment:
Please, use "error first" approach. https://www.geeksforgeeks.org/error-first-callback-in-node-js/